### DISCUSSION TITLE
visibility elevation profile

### DISCUSSION THREAD

**Discussion by janverhelst13** [osmandapp/OsmAnd#18489] [Ideas]
As a cyclist, the elevation and gradient profile are very interesting. But you can´t see the first part clearly as there are some figures of the axes in front of it. Is it possible to charge that? That the graph itself starts after the figures on the axes?
<table style="width:100%">
  <tr>
<th><img src="https://github.com/osmandapp/OsmAnd/assets/149667624/21eb5296-5ff4-4b07-b299-3852e7ab466f"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by mikehgentry** [comment]
Duplicate of #18204 (via #18402)
